docs/release-notes/2.29.0.md

```diff
@@ -0,0 +1,215 @@
+- More details and targets in commands 
+- Ppof data collection: Obtain profiling data from the PMM server with the pprof parameter in the pmm-admin summary enabling you to generate reports in textual and graphical formats for visualizing and analyzing data.
+
+
+### Percona Platform Integration
+
+#### Terminate Percona Platform connections as an Admin
+When regular users are not able to disconnect a server (for example when PMM was moved to a network segment without outbound connections to public networks), PMM Admins can now terminate such connections instead.
+
+The **Disconnect** option under **Settings > Percona Platform** now enables PMM Admins to retire PMM instances from Percona Platform even if they are not logged in with a Percona Account.
+
+When disconnecting servers without a Percona Account, the lingering servers are not automatically removed from the list of PMM instances in Percona Platform. As a Percona Platform Admin, you can ensure that your list of PMM instances stays up-to-date by manually removing those unavailable servers via [PMM instances](https://portal.percona.com/login).
+
+#### Easier access to your Percona Support Specialist 
+If you are a Percona customer, you can now find the contact details of your dedicated Percona Support Specialist on the new ![!clouds](../_images/clouds.png) **Environment Overview** tab on the main menu.
+
+The **Percona Contacts** section here shows the name and email of the Customer Success Manager assigned to your organization, who can help with any PMM queries.
+This new tab will soon be populated with more useful information about your PMM environment.
+
+
+### Advisors
+#### Extended severities for advisor checks and alerts
+We have extended and unified the list of severity levels for the notifications coming from Integrated Alerting and Advisor Checks. 
+When you create a new alert rule, you can now choose from a wider range of severity labels: **Emergency**, **Alert**, **Critical**, **Error**, **Warning**, **Notice**, **Info**, **Debug**. 
+
+The same severities are now also available for notifications coming from failed checks under **Advisor Checks > Failed Checks**. We’ve also made a small UX improvement to the way failed checks are grouped by severity in the table.
+
+#### Use filters to explore available advisors checks
+As we’re constantly adding more advisors for checking the health of your connected databases, you may find it useful to now be able to drill-down through this constantly growing list.    
+
+Use the filters on the **Advisors Checks > All Checks** page to search advisors by **Name**, **Description**, **Status** or **Interval**. If you need to share the filtered results with your team members, send them the PMM URL which saves your search criteria and results. 
+
+### DBaaS
+
+Simplified the API such that it requires fewer parameters to run the API. Also, enhanced the documentation.
+
+### Exporters
+
+Updated the MySQL exporter.
```

Comment:
Exporters are not independent features, so this is part of the "Components update." But we also need to add a value to this action .. what it gives users